utils/Cargo.toml

```diff
@@ -19,3 +19,6 @@ criterion.workspace = true
 codespan.workspace = true
 serde = { workspace = true, features = ["derive"] }
 toml = { workspace = true, features = ["parse"] }
+
+[target.'cfg(target_family = "unix")'.dependencies]
+pprof = { workspace = true, features = ["criterion", "flamegraph"] }
```

Comment:
Can this be a dev-dependency?

Reply:
It must be a dependency because this is a library. I initially used dev-dependency on the other PR and it caused CI failures.

Reply:
Ah, right. But it should be possible to make it optional (i.e. depend on a non-default feature flag) and then only bring in that feature in the dev-dependencies of nickel-lang-core?

Reply:
I've made that change here: https://github.com/tweag/nickel/pull/2025/commits/0fbcf71ba24526998164c4629a40ead59870403d